### ISSUE TITLE
lit-analyzer 2.0.0-pre.1 - Error: Cannot find module './lib/index.cjs'

### ISSUE THREAD

**Issue by web-padawan** [runem/lit-analyzer#258]
When trying to run `lit-analyzer@next` the following error is thrown: 

```sh
$ npx lit-analyzer frontend/**/*.ts
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './lib/index.cjs'
Require stack:
- /Users/serhii/vaadin/docs/node_modules/lit-analyzer/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/serhii/vaadin/docs/node_modules/lit-analyzer/cli.js:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/serhii/vaadin/docs/node_modules/lit-analyzer/cli.js' ]
```

As you can see here, the file imported is not published to npm:

<img width="287" alt="Screenshot 2022-05-09 at 18 22 59" src="https://user-images.githubusercontent.com/10589913/167442917-e12fa8b4-e2c4-4837-8672-db8611fb0855.png">

@rictic @justinfagnani FYI, possibly related to recent build changes.

**Comment by rictic** [comment]
Yep, that looks like a config error, will take a look.

**Comment by rictic** [comment]
Published lit-analyzer@2.0.0-pre.2 which should fix this
